folding/validators/forward.py

```diff
@@ -7,7 +7,7 @@
 from typing import List, Dict
 
 from folding.validators.protein import Protein
-from folding.utils.logging import log_event
+from folding.utils.folding_log import log_event
```

Comment:
this shouldn't work because it's a method of the class

Reply:
https://github.com/macrocosm-os/folding/pull/147/commits/3adc14eae43552a0e3bbba5ea3397be5c37543d2